fem/gridfunc.cpp

```diff
@@ -4634,4 +4634,801 @@ GridFunction *Extrude1DGridFunction(Mesh *mesh, Mesh *mesh2d,
    return sol2d;
 }
 
+void GridFunction::SetupPLBounds(const int nb, const int ncp,
```

Comment:
Do you think the check for tensor-product elements should be here at setup time instead of at bound eval time?